tests/end_to_end/workflow/include_exclude_flow.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from openfl.experimental.workflow.interface import FLSpec
+from openfl.experimental.workflow.placement import aggregator, collaborator
+
+log = logging.getLogger(__name__)
+
+class TestFlowIncludeExclude(FLSpec):
```

Comment:
Query: Same comment as in TestflowExclude. Wondering the reason to modify the flow to remove _include_exclude_error_list_ which could potentially help with debugging